doc/classes/ProjectSettings.xml

```diff
@@ -2482,7 +2488,7 @@
 			During each physics tick, Godot will multiply the linear velocity of RigidBodies by [code]1.0 - combined_damp / physics_ticks_per_second[/code], where [code]combined_damp[/code] is the sum of the linear damp of the body and this value, or the area's value the body is in, assuming the body defaults to combine damp values. See [enum RigidBody2D.DampMode].
 			[b]Warning:[/b] Godot's damping calculations are simulation tick rate dependent. Changing [member physics/common/physics_ticks_per_second] may significantly change the outcomes and feel of your simulation. This is true for the entire range of damping values greater than 0. To get back to a similar feel, you also need to change your damp values. This needed change is not proportional and differs from case to case.
 		</member>
-		<member name="physics/2d/physics_engine" type="String" setter="" getter="" default="&quot;DEFAULT&quot;">
+		<member name="physics/2d/physics_engine" type="String" setter="" getter="" default="&quot;DEFAULT&quot;" keywords="godotphysics">
```

Comment:
While this is a relatively innocent addition, I struggle to imagine anyone looking for `godotphysics` specifically. It's Godot's own barebones engine that, if it weren't listed, few would think it even has a name other than "Default", especially because it doesn't have its own exclusive section of settings (unlike Jolt). I know a few people that haven't even messed with these physics settings and stuck to the default.